test/emqx_ocsp_cache_SUITE.erl

```diff
@@ -291,6 +307,11 @@ get_sni_fun(ListenerID) ->
     SSLOpts = proplists:get_value(ssl_options, Opts),
     proplists:get_value(sni_fun, SSLOpts).
 
+openssl_version() ->
+    "OpenSSL " ++ Res = os:cmd("openssl version"),
```

Comment:
this might not work on macos, e.g. here what I get on apple silicon & monterey:
```bash
$ openssl version
LibreSSL 2.8.3
```

not sure how much we should care about macos shenanigans here, just FYI

Reply:
Thanks for the heads up!  :beers: 

I'll then assume the first word is the program name, that'll hopefully be flexible enough :crossed_fingers: